### ISSUE TITLE
support for Honeywell dolphin ct50 

### ISSUE THREAD

**Issue by pramodwerea** [tianhsky/honeywell_scanner_plugin#1]
Hi does this plugin works for Honeywell dolphin ct50 version?

**Comment by andreujuanc** [comment]
I will test this in 15 days. Let me know if you find something before that. Cheers

**Comment by pramodwerea** [comment]
Thanks, @andreujuanc, I will wait for your feedback and if possible will try to work at my end and will update if I manage to find something.

**Comment by pramodwerea** [comment]
Hi @andreujuanc, have you tried it for Honeywell dolphin ct50?

**Comment by andreujuanc** [comment]
@pramodwerea Mate, sorry I haven't answer you yet but they send me the wrong device, and I have to send it back 👎 Was a miscommunication between my company and their local representative.
Have you tried?

**Comment by pramodwerea** [comment]
@andreujuanc not yet, since I was busy with some other project. But now we need to start this as well so I was thinking if you have tried and was waiting for your feedback. It will be great if you can do it for this device as well. You think, you can try in coming days soon?

**Comment by andreujuanc** [comment]
@pramodwerea Ill do my best.

**Comment by pramodwerea** [comment]
@andreujuanc Thanks a lot for your quick response and support, will be waiting for your feedback!

**Comment by andreujuanc** [comment]
@pramodwerea What version of android are you planing to support?  I think there was another project around aswel. But I cant test without my device.
Please let me know if you find anything somewhere!

**Comment by pramodwerea** [comment]
@andreujuanc we are thinking for Android 4+. Do you have honeywell dolphin ScanPal EDA50. We have these two devices in option to use, so if you can test with this, that will be also helpful.

which is the other project that support these devices?

**Comment by andreujuanc** [comment]
@pramodwerea No, i don't have those. We will only work on CT50. I did a small bridge for Windows Universal Platform. Works quite well. https://github.com/andreujuanc/cordova-plugin-pointofservice

Have you tried this? https://github.com/andreujuanc/cordova-honeywell

**Comment by pramodwerea** [comment]
@andreujuanc  thanks, and will update after test!

**Comment by pramodwerea** [comment]
Hi today I tried https://github.com/andreujuanc/cordova-honeywell and it works for EDA50 Honeywell. But it does not support QR code. Is there any option to scan for QR code as well.

**Comment by andreujuanc** [comment]
@pramodwerea  Thanks for feedback, I still can't test on android. I'm waiting for honeywell to update my device :(

**Comment by tianhsky** [comment]
What error did you get? I think you can configure honeywell device to enable QRcode scanning.

**Comment by pramodwerea** [comment]
@tianhsky while scanning QRcode, neither it triggers error nor enters into listenForScans. After releasing scan button just returns scan fail. QRcode is enabled in the device because the native scan is fine.

**Comment by andreujuanc** [comment]
@tianhsky I'm starting to test everything now.
Did you tried this project? What both projects are lacking is options. 
Will see to it.
EDIT: Forgot to tell you, meanwhile you can go to settings > Scanning > Internal Scanner > Default Profile > Symbology Settings and choose what you app going to use. 
RESTART DEVICE AFTER CHANGES.

**Comment by andreujuanc** [comment]
Ill be updating my fork to allow symbologies, also will work on CT50 Windows 10 Mobile.

**Comment by andreujuanc** [comment]
@pramodwerea Worked! Worked on my fork, but i reverted the changes I made, so you can just grab the original.
Unless you want to use the same symb as I have.
